File: Selection sort.py
size=int(input('Enter the size of your array: '))
l=list(map(int,input('Enter your array: ').split()))
for i in range(size-1):
    pos=i
    for j in range(i+1,size):
        if l[pos]>l[j]:
            pos=j
    l[pos],l[i]=l[i],l[pos]
print('Sorted array: ',end='')
for i in l:
    print(i,end=' ')

File: In_c/Selection sort.c
#include<stdio.h>
void swap(int*,int*);
int main(){
    int n,arr[100],i,j,pos;
    printf("Enter size of your array: ");
    scanf("%d",&n);
    printf("Enter elements: ");
    for(i=0;i<n;i++)
            scanf("%d",&arr[i]);
    for(i=0;i<n-1;i++){
        pos=i;
        for(j=i+1;j<n;j++)
            if(arr[pos]>arr[j])
                pos=j;
        swap(&arr[i],&arr[pos]);
    }
    printf("Sorted array: ");
    for(i=0;i<n;i++)
        printf("%d ",arr[i]);
}
void swap(int *a,int *b){
    int tmp=*a;
    *a=*b;
    *b=tmp;
}
